src/quo2/components/settings/reorder_item/items/item_tabs.cljs

```diff
@@ -0,0 +1,41 @@
+(ns quo2.components.settings.reorder-item.items.item-tabs
+  (:require [quo2.components.tabs.segmented-tab :as quo2]
+            [quo.react-native :as rn]
+            [quo.components.text :as text]
+            [quo2.components.settings.reorder-item.style :as style]
+            [quo2.components.icon :as quo2-icons]))
+
+(defn render-tab-item
+  [item]
+  (let [tab-image (cond
+                    (item :image) [rn/image
+                                   {:source (:image item)
+                                    :style  style/tab-item-image}]
+                    (item :icon)  [rn/view {:style style/tab-item-image}
+                                   (quo2-icons/icon (:icon item) (style/tab-icon))])]
+    [rn/view
+     {:style style/tab-item-container}
+     tab-image
+     [text/text
+      {:style style/tab-item-label
+       :width :medium}
+      (:label item)]]))
+
+(defn transform-data
+  [data]
+  (map #(hash-map :id (:id %) :label (render-tab-item %)) data))
+
+(defn view
+  [{:keys [data default-active on-change]
+    :or   {data           nil
+           default-active 1
+           on-change      (fn [] nil)}}]
```

Comment:
`(constantly nil)` or `#()` seems to be better (I prefer `(constantly nil)`)

Reply:
Done